nifi-nar-bundles/nifi-box-bundle/nifi-box-services/src/main/resources/docs/org.apache.nifi.box.controllerservices.JsonConfigBasedBoxClientService/additionalDetails.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="en">
+<!--
+  Licensed to the Apache Software Foundation (ASF) under one or more
+  contributor license agreements.  See the NOTICE file distributed with
+  this work for additional information regarding copyright ownership.
+  The ASF licenses this file to You under the Apache License, Version 2.0
+  (the "License"); you may not use this file except in compliance with
+  the License.  You may obtain a copy of the License at
+      http://www.apache.org/licenses/LICENSE-2.0
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+-->
+<head>
+    <meta charset="utf-8" />
+    <title>ListBoxFiles</title>
+
+    <link rel="stylesheet" href="../../../../../css/component-usage.css" type="text/css" />
+</head>
+<body>
+
+<h1>Setting up a Box App</h1>
+
+<p>This processor requires a pre-configured App under the Account owning the resources being accessed.</p>
+<p>The App should have the following configuration:</p>
+<ul>
+    <li>
+        If you create a new App, select 'Server Authentication (with JWT)' authentication method.
+        <br/>If you want to use an existing App, chose one with 'OAuth 2.0 with JSON Web Tokens (Server Authentication)' as Authentication method.
+    </li>
+    <li>Should have a 'Client ID' and 'Client Secret'.</li>
+    <li>'App Access Level' should be 'App + Enterprise Access'.</li>
+    <li>'Application Scopes' should have 'Write all files and folders in Box' enabled.</li>
+    <li>'Advanced Features' should have 'Generate user access tokens' and 'Make API calls using the as-user header' enabled.</li>
+    <li>
+        Under 'Add and Manage Public Keys' Generate a Public/Private Keypair and download the configuration JSON file (under App Settings). The full path of this file should be set in the 'Box Config File' property.
+        <br/>Note that you can only download the configuration JSON with the keypair details only once, when you generate the keypair. Also this is the only time Box will show you the private key.
+        <br/>If you want to download the configuration JSON file later (under 'App Settings') - or if you want to use your own keypair - after you download it you need to edit the file and add the keypair details manually.
+    </li>
+    <li>After all settings are done, the App needs to be reauthorized. ('Reauthorize App' on the https://app.box.com/master/custom-apps page.)</li>
+</ul>
```

Comment:
When the app is configured for the first time, it needs to be authorized which can be requested on the app's page, Authorization tab pressing "Review and Submit" button. 
The app is not listed on https://app.box.com/master/custom-apps before the first authorization is made. 

I'd keep the note about reauthorization since it's good to know that an authorized app needs to be reauthorized every time an app setting is changed.